lib/handlers/handler.js

```diff
@@ -11,6 +11,7 @@ function handler(info, params, res) {
       res.json({
         ok: false,
         error: (error && error.message) || error,
+        details: (error && error.details) || undefined,
```

Comment:
Random q - If this is undefined, does it still render the property? Or will the result object only have a 'details' key if there is actual content